src/azure-cli/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -6,6 +6,7 @@
 # pylint: disable=line-too-long
 # pylint: disable=too-many-statements
 # pylint: disable=too-many-locals
+# pylint: disable=all
```

Comment:
Pls don't disable all pylint............

You can add specific pylint disable rule in specific line

Reply:
corrected!